packages/components/sort-select/stories/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+import { useArgs } from '@storybook/preview-api';
+
+/**
+ * Internal dependencies
+ */
+import SortSelect, { type SortSelectProps } from '..';
+
+export default {
+	title: 'Checkout Components/SortSelect',
```

Comment:
I would suggest renaming this section "External Components" based on our chats so far, as not all components here will be Checkout related.

Reply:
Sounds good actually. I like that.